source/extensions/tracers/zipkin/zipkin_tracer_impl.h

```diff
@@ -23,6 +23,7 @@ namespace Zipkin {
 #define ZIPKIN_TRACER_STATS(COUNTER)                                                               \
   COUNTER(spans_sent)                                                                              \
   COUNTER(timer_flushed)                                                                           \
+  COUNTER(reports_skipped)                                                                         \
```

Comment:
Given the only way this can happen now is no cluster, I would probably make this more clear: `reports_skipped_no_cluster` or similar. Same elsewhere.

Reply:
Fixed